### PR TITLE
add mastodon shorturls

### PR DESCRIPTION
**Pull request by MrMinos**

### CODE REVIEW

File: modules/ocf_www/manifests/site/shorturl.pp [hunk 1]
```diff
@@ -110,6 +111,8 @@
       {rewrite_rule => '^/s/(.+)$ https://www.ocf.berkeley.edu/api/shorturl/$1 [R]'},
       {rewrite_rule => '^/stf-cost-breakdown$ https://docs.google.com/spreadsheets/d/1U3YfU5S1hyi4c9u1vME84lz5hQPz23LKWCaD8mq4JHI/edit [R]'},
       {rewrite_rule => '^/survey$ https://goo.gl/forms/sk1s0dnuxDY71vBR2 [R]'},
+      {rewrite_rule => '^/thedon$ https://mastodon.ocf.berkeley.edu [R]'},
+      {rewrite_rule => '^/thedong$ https://mastodon.ocf.berkeley.edu [R]'},
```
Comment (abizer) [inline modules/ocf_www/manifests/site/shorturl.pp]: delet this nephew

Comment (abizer) [inline modules/ocf_www/manifests/site/shorturl.pp]: delet this nephew

Comment (MrMinos) [reply modules/ocf_www/manifests/site/shorturl.pp]: 😞

**Comment by cg505** [comment]
This is fine, but you can just do mastodon.ocf.io which is just as long as ocf.io/mastodon.

**Comment by MrMinos** [comment]
> Please add `mastodon` to `RESERVED_USERNAMES` in [this file](https://github.com/ocf/ocflib/blob/master/ocflib/account/validators.py).

Done~ https://github.com/ocf/ocflib/pull/173

**Pull request merged**